databases/week2/LibraryDatabase.sql

```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE LibraryDatabase;
+
+USE LibraryDatabase;
+
+CREATE TABLE `Member` (
+  `Member_ID` int unsigned NOT NULL AUTO_INCREMENT,
+  `First_Name` varchar(255) NOT NULL,
+  `Last_Name` varchar(255) NOT NULL,
+  `Address` varchar(255) NOT NULL,
+  `Email` varchar(255) NULL,
+  `Phone` varchar(20) NULL
+); 
+
+CREATE TABLE `Book_Details` (
+  `Book_ID` int unsigned NOT NULL AUTO_INCREMENT,
+  `Title` varchar(255) NOT NULL,
+  `Author` varchar(255) NOT NULL,
+  `Genre` varchar(255) NOT NULL,
+  `Shelf_ID` int NOT NULL,
+  `Status_ID` int NOT NULL,
+  `Reserved` varchar(20) NOT NULL
+); 
+
+CREATE TABLE `Loaned_Books` (
+  `Index_ID` int unsigned NOT NULL AUTO_INCREMENT,
+  `Member_ID` int unsigned NOT NULL,
+  `Book_ID` int unsigned NOT NULL,
+  `Borrow_Date` DATETIME NOT NULL,
+  `Due_Date` DATETIME NOT NULL,
```

Comment:
```suggestion
  `Due_Date` DATETIME NOT NULL,
  CONTRAINT 'fk_member' FOREIGN KEY ('member_id') REFERENCES 
  'Member'('member_id') ON DELETE CASCADE ON UPDATE CASCADE,
  CONTRAINT 'fk_book' FOREIGN KEY ('Book_ID') REFERENCES 'Book Details'('BookID') ON DELETE CASCADE ON UPDATE CASCADE 
```

Reply:
I get a lot of errors when I run foreign keys, but yes, clearly I have to spend more time on this subject so I will redo the homework.

Reply:
If possible I would recommend bringing this issue to class, it is easier to help you with the error messages in person.